src/Services/QueuedJobService.php

```diff
@@ -66,6 +66,37 @@ class QueuedJobService
      */
     private static $stall_threshold = 3;
 
+    /**
+     * How early stuck jobs will become eligible for automated retry processing (minutes)
+     *
+     * @config
+     * @var int
+     */
+    private static int $job_retry_sentinel = 1;
+
+    /**
+     * How many jobs are eligible to be automatically retried per one health check
+     * Setting this to 0 disables the job retry feature
+     *
+     * @config
+     * @var int
+     */
+    private static int $job_retry_limit = 0;
+
+    /**
+     * Defines the job status transformation for job retries
+     * Set rule value to "null" to disable it
+     *
+     * @config
+     * @var array
+     */
+    private static array $job_retry_status = [
```

Comment:
```suggestion
    private static array $job_retry_status_map = [
```
This doesn't represent a single status - but a map of statuses.